py3status/modules/gpmdp.py

```diff
@@ -4,55 +4,125 @@
 
 Configuration parameters:
     cache_timeout:  refresh interval for this module (default 5)
-    format:         specify the items and ordering of the data in the status bar.
-                    These area 1:1 match to gpmdp-remote's options
-                    (default '♫ {info}')
+    format:         display format for this module (default '♫[ {status}]')
+    state_paused:   show this when gpmdp paused playback
+        (default '[(Paused) {song_title} - {song_artist}]')
+    state_playing:  show this when gpmdp started playback
+        (default '{song_title} - {song_artist}')
```

Comment:
I went through it again and made few changes. I converted useless `123456` milliseconds to `02:14` thing because that's needed. I feel this is ready to go in, be reviewed, etc.
End of comment.

New comment. Things that occurs to me whilst playing with modules...
```
      cache_timeout: refresh interval for this module (default 2)
 +    catnip_timeout: refresh interval for this module (default 60)
```

Say, I don't always play`gpmdp`, `cmus`, `clementine`, `mpris`, `spotify`, `deadbeef`, `player_control`, etc... often so this default timeout might seem silly (2 sec) (or 5 sec) nonstop for most of them. Other modules are like that too. We're only using one `cache_timeout` for almost everything. `catnip_timeout` goes right under `cache_timeout`.

Is it more efficient to place certain `catnip` in some modules? If `gpmdp` is not started, use `catnip_timeout (60)` until it's started, then `cache_timeout (2)`. If `60` is too much, maybe `30` or `15`? `15` is still better than `2*7`. Thoughts? Useful or useless optimization?